source/linux/util.js

```diff
@@ -4,6 +4,25 @@ import {promises as fsPromises} from 'node:fs';
 
 export const execFile = promisify(childProcess.execFile);
 export const exec = promisify(childProcess.exec);
+export const spawn = (
+	cmd,
+	args,
+) => new Promise(resolve => {
+	const cp = childProcess.spawn(cmd, args);
+	cp.on('spawn', () => {
+		resolve();
+	});
+
+	cp.on('exit', (code, signal) => {
+		if (code) {
+			console.log(`swaybg exited with ${code} code`);
+		}
+
+		if (signal) {
+			console.log(`swaybg exited with ${signal} signal`);
+		}
```

Comment:
This should be properly handled with `reject`/`resolve` and not log.

Reply:
I thought about that then I realized that swaybg can't exit from invalid argument and the only way that user stop swaybg(accidentally or not) is writing wrong flag or killing it by himself

Reply:
So I just removed that part of code